src/software/parameter/config/simulator.yaml

```diff
@@ -0,0 +1,25 @@
+SimulatorConfig:
+  SlidingFrictionAcceleration:
+    min: 0
+    max: 10
+    default: 0.0
```

Comment:
like @kelvinkoon comment, wouldn't sliding friction being defaulted to 0 be problematic?

Reply:
added a comment